packages/core/config/storybook/main.js

```diff
@@ -1,13 +1,16 @@
 const path = require('path')
 const fs = require('fs')
 
-const { getPaths } = require('@redwoodjs/internal')
+const { getConfig, getPaths } = require('@redwoodjs/internal')
 const { getSharedPlugins } = require('../webpack.common')
 
+const config = getConfig()
+
 module.exports = {
   stories: [
     '../../../../web/src/**/*.stories.{tsx,jsx,js}',
   ],
+  addons: config.web.a11y ? ['@storybook/addon-a11y'] : undefined,
```

Comment:
Not convinced that this is the best way to conditionally add a property? Just need to make sure that having `addons` as `undefined` isn't actually breaking anything...

Reply:
If it doesn't like `undefined` (which shouldn't be a problem) another option to try is just an empty array
```
  addons: config.web.a11y ? ['@storybook/addon-a11y'] : [],
```

Another option is to spread it

```
module.exports = {
  stories: [
    '../../../../web/src/**/*.stories.{tsx,jsx,js}',
  ],
  ...(config.web.a11y && { addons: ['@storybook/addon-a11y'] }),
}
```

Reply:
@Tobbe you posted this in the #TIL! This is super cool to know